vue.config.js

```diff
@@ -11,12 +11,21 @@ module.exports = {
   },
   chainWebpack: (config) => {
     config.devtool('cheap-module-source-map');
+    config.resolve.alias.set('vue-i18n', 'vue-i18n/dist/vue-i18n.runtime.esm-bundler');
     config.module
       .rule('i18n')
       .test(/\.(json5?|ya?ml)$/)
       .type('javascript/auto')
       .use('i18n')
-      .loader('@intlify/vue-i18n-loader');
+      .loader('@intlify/vue-i18n-loader')
+      .end();
+    config.module
+      .rule('vue')
+      .use('vue-loader')
+      .tap((options) => ({
+        ...options,
+        refSugar: true
```

Comment:
ref sugar seems not working. $ref is not defined is reported.

Reply:
ref sugar is working on my computer. Maybe you have to run yarn again to install the latest deps